Hangman/Program.cs

```diff
@@ -0,0 +1,240 @@
+﻿using System;
```

Comment:
Comments explaining what the classes represent and what the methods do would have been nice.